src/test/java/net/sf/jabref/logic/citationstyle/CitationStyleTest.java

```diff
@@ -0,0 +1,14 @@
+package net.sf.jabref.logic.citationstyle;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class CitationStyleTest {
+
+    @Test
+    public void getDefault() throws Exception {
+        Assert.assertNotNull(CitationStyle.getDefault());
+    }
+
```

Comment:
Please add some tests. At least one test should verify that a BibEntry is properly converted into the correct string.